### PR TITLE
Updating README to clarify Step 3

### PR DESCRIPTION
**Pull request by Jxsephh**
Added some clarification to the `README` that expands on some of the steps needed to configure the server and initialize the database using `psql` with Windows, which has some additional steps when installing for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,12 +135,21 @@ To set up and launch the server application:
...
-1. Update the newly created `.env` file and update the `DB_HOST`, `DB_USERNAME`, `DB_PASSWORD`, `DB_PORT` and `DB_DATABASE_NAME` with the values from the credentials you obtained in [Step 1](#1-Provision-a-PostgreSQL-instance). If you created a certificate file in the previous action, then also update the `DB_CERTFILE` with the location of this file (relative to the `server` directory).
-1. Also update the `CMS_USERNAME`, `CMS_PASSWORD`, `CLIENT_ID` and `CLIENT_SECRET` with the values from creating your instance of Watson Media, from [Step 2](#2-Set-up-an-instance-of-Watson-Media).
-1. Prepare to initialize the database with the correct tables. Scripts are provided that do this using the `psql` cli, which is recommended that you install. On macOS, for instance, you can do this with the brew command:
-    - `brew install libpq`
-    - You may also like to link the `psql` command to you local bin directory with brew `link --force libpq`
+### 3. Configuring and running the server
```
Comment (markstur) [inline README.md:138]: Now there are two "3. Configuring and running the server" headings.  Looks like a mistake.

**Review by markstur — CHANGES_REQUESTED** [review]
There's a duplicate header.  See inline comment.
Otherwise it looked OK, but seeing "3. Configuring and running the server" twice definitely looks wrong.

File: README.md (hunk 1 above)
Comment (Jxsephh) [reply README.md]: You are correct, my mistake! Thank you for catching!

File: README.md [hunk 2]
```diff
@@ -135,12 +135,20 @@ To set up and launch the server application:
...
+1. Update the newly created `.env` file and update the `DB_HOST`, `DB_USERNAME`, `DB_PASSWORD`, `DB_PORT` and `DB_DATABASE_NAME` with the values from the credentials you obtained in [Step 1](https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop#1-Provision-a-PostgreSQL-instance). If you created a certificate file in the previous action, then also update the `DB_CERTFILE` with the location of this file (relative to the `server` directory). For example, `DB_CERTFILE=./cert.pem`.
+1. Also update the `CMS_USERNAME`, `CMS_PASSWORD`, `CLIENT_ID` and `CLIENT_SECRET` with the values from creating your instance of Watson Media, from [Step 2](https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop#2-Set-up-an-instance-of-Watson-Media). 
+
+ 1. Prepare to initialize the database with the correct tables. Scripts are provided that do this using the  `psql`  CLI, which is recommended that you install:
+	- macOS:	
+        -   `brew install libpq`
+        -   You may also like to link the  `psql`  command to you local bin directory with brew  `link --force libpq`
+
+	- Windows:
+        - Download PostgreSQL for Windows [here]. (https://www.postgresql.org/download/windows/)
```
[the remark this answers is not in the document]
Comment (markstur) [reply README.md]: fixed

**Review by markstur — APPROVED** [review]
Thanks!  Looks good now.  I'm a little unsure about installing all of postgres here instead of just the client piece, but perhaps it is better than a Mac-only readme. Also there is another PR suggesting the full local install alternative so maybe this will come together better.

Is there a download for just the client on Windows?  I don't see one.

**Comment by Jxsephh** [comment]
> Thanks! Looks good now. I'm a little unsure about installing all of postgres here instead of just the client piece, but perhaps it is better than a Mac-only readme. Also there is another PR suggesting the full local install alternative so maybe this will come together better.

Thanks for reviewing! I hadn't done much markdown before, so I apologize for the errors.

> Is there a download for just the client on Windows? I don't see one.

 Great question, [this](https://stackoverflow.com/questions/33854798/how-do-i-install-just-the-client-tools-for-postgresql-on-windows) StackOverflow answer says that there isn't one but details what to remove in order to trim the fat on the download. I don't mind adding those steps to the `README` as an optional step, if you'd like!

**Comment by markstur** [comment]
No problem re: the markdown tips. That's what reviewers are for.

re: the Windows install, I'm leaning toward assuming they might find "the fat" useful if that's the way it comes.  But if you think it's better to mention a link that's fine too. Try to keep it simple though.  I think once the other PR lands, folks might be likely to run a local postgres anyway and keep that fat.

**Pull request merged**